packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts

```diff
@@ -68,6 +69,14 @@ export class TestsSelector
       };
     });
 
+    fileItems.push({
```

Comment:
Nit - do we want to keep `Run All Tests` at the top of the picklist and move `Run All Local Tests` to the second spot? That way the picklist is arranged from general -> specific options to run tests

Reply:
That was actually my original change. I'm arranging it this way since running local tests is default for production deployments, and it's the behavior of "Run All Tests" with library toggle turning off (Before the changes introduced in this PR). I'd like the first/default option to be more likely/frequently the option that customers would be choosing.

Reply:
Cool that makes sense to me as well